app/views/application/tips.html.erb

```diff
@@ -0,0 +1,85 @@
+<main>
+  <%= render "header" %>
+</main>
+<center>
+<h2 style="padding:10px">
+How to Bird
+</h2>
+</center>
+<p style="padding:80px; text-indent:50px">
+
+Location, time of day, weather, and time of year can influence what types and numbers of birds you might observe.
+
+Get to know your regulars. Some birds in your neighborhood are common, year-round residents. By learning to identify these birds by sight and sound, and by recognizing their habits (How do they move? What do they eat? What do they sound like?) you will better be able to notice when something unusual comes along.
```

Comment:
If these are supposed to be paragraphs, make sure theyre wrapped in P tags. I would think they are, since otherwise this is a HUGE block of text and that is harder to read.